integration_tests/steps/integration_tests.py

```diff
@@ -214,8 +214,10 @@ def one_sided_market(context, market_type):
         ConstSettings.IAASettings.MARKET_TYPE = 1
     elif market_type == "two-sided-pay-as-bid":
         ConstSettings.IAASettings.MARKET_TYPE = 2
-    if market_type == "two-sided-pay-as-clear":
-        ConstSettings.IAASettings.MARKET_TYPE = 3
+        ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = 2
```

Comment:
This is wrong right? It should be 1 or:
```suggestion
        ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = BidOfferMatchEnum.PAY_AS_BID.value
```

Reply:
Maybe the reason for the reduced SS/CS?

Reply:
Don't think so as the tageted `kpi integration test` explicitly targeted PAB